### PR TITLE
feat(emails): update post add recovery key email

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - new copy for the post add recovery key email

This commit:
 - update email templates for post add recovery key email

### CODE REVIEW

File: packages/fxa-auth-server/lib/senders/emails/templates/postAddAccountRecovery/en.ftl [hunk 1]
```diff
@@ -1,6 +1,9 @@
-postAddAccountRecovery-subject = Account recovery key generated
-postAddAccountRecovery-title = Account recovery key generated
-postAddAccountRecovery-description = You have successfully generated an account recovery key for your { -product-firefox-account } using the following device:
+postAddAccountRecovery-subject-2 = Account recovery key created
+postAddAccountRecovery-title2 = You created a new account recovery key
+postAddAccountRecovery-description-2 = A new key was created from:
```
Comment (bcolsson) [inline packages/fxa-auth-server/lib/senders/emails/templates/postAddAccountRecovery/en.ftl:3]:
```suggestion
# Information on the browser and device triggering this string follows.
postAddAccountRecovery-description-2 = A new key was created from:
```
Suggest adding a comment for context.

**Review by xlisachan — APPROVED** [review]
r+wc!

**Comment by chenba** [comment]
@xlisachan Thank you!

**Comment by chenba** [comment]
@bcolsson I need an approval from you in order to merge this.  Thanks!

**Review by bcolsson — APPROVED** [review]

**Pull request merged**